src/figures.jl

```diff
@@ -165,7 +170,12 @@ end
 # Layouts are already hooked up to this, so it's very simple.
 """
     resize!(fig::Figure, width, height)
+    resize!(width, height)
+
 Resizes the given `Figure` to the resolution given by `width` and `height`.
 If you want to resize the figure to its current layout content, use `resize_to_layout!(fig)` instead.
+
+If no figure is provided, `fig` defaults to `current_figure()`.
 """
 Makie.resize!(figure::Figure, args...) = resize!(figure.scene, args...)
+Makie.resize!(args...) = resize!(current_figure().scene, args...)  # Less specific type, so definition above is used if figure is passed.
```

Comment:
This is actually `Base.resize!`, so providing this general definition is probably not a good idea.

Reply:
Huh. So this is actually type piracy? I'll remove the line.

Reply:
Yeah, looks like it...